app/views/occupation_standards/_similar_results_list.html.erb

```diff
@@ -0,0 +1,59 @@
+<script>
+  function changeVisibility(element_id) {
+
+       var panel = document.getElementById(element_id);
+       var panelButton = document.getElementById(`${element_id}-button`);
+       if (panel.style.display === "flex") {
+         panelButton.style.display = "flex";
+         panel.style.display = "none";
+         panel.setAttribute("aria-expanded", false)
+       } else {
+         panelButton.style.display = "none";
+         panel.style.display = "flex";
+         panel.setAttribute("aria-expanded", true)
+       }
+   }
+</script>
+
+<% similar_results = occupation_standard.similar_programs %>
+
+<% if similar_results.count > 0 %>
+  <div id="accordion-<%= occupation_standard.id %>-button" class="mt-4 flex flex-col space-x-4 lg:space-x-0 items-center rounded bg-gray-100 py-3 px-2 text-sm lg:flex-row">
+    There <%= "is".pluralize(similar_results.count, :en) %> <%= pluralize(similar_results.count, "program") %> with similar or identical criteria.
+    <button aria-controls="accordion-<%= occupation_standard.id %>" class="font-bold underline lg:mx-1" href="#" onclick='changeVisibility("accordion-<%= occupation_standard.id %>")'>Expand duplicates</button>
+  </div>
+  <div id="accordion-<%= occupation_standard.id %>" style="display: none;" class="mt-4 similar-results-box bg-gray-100 py-3 px-2 flex flex-col">
```

Comment:
Can we use the tailwind `hidden` class here instead of `style="display: none:"`?

Reply:
I'm using javaScript to set display. I wasn't sure about doing it with tailwind. I can look into it

Reply:
No biggie if it makes it complicated.

Reply:
I switched to toggling the hidden class. I'm still working on getting the stimulus controller setup. I'm also getting an rspec failure in Github that I'm not getting locally. It doesn't seem related to my changes.